e2e/various-suite/return-to-previous.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { e2e } from '../utils';
+
+describe('ReturnToPrevious button', () => {
+  beforeEach(() => {
+    e2e.flows.login(Cypress.env('USERNAME'), Cypress.env('PASSWORD'));
+    cy.window().then((win) => {
+      win.localStorage.setItem('grafana.featureToggles', 'returnToPrevious=1');
+    });
+
+    cy.visit('/alerting/list?search=');
+    e2e.components.AlertRules.groupToggle().first().click();
+    e2e.components.AlertRules.toggle().click();
+    e2e.components.AlertRules.expandedContent().find('[data-testid="data-testid go to dashboard"]').click();
+    // TODO: replace data-testid in find?
+  });
+
+  it('should appear when changing context and go back to alert rule when clicking "Back"', () => {
+    // make sure the dashboard finished loading
+    cy.get('button[aria-label*="BarChart - Label Rotation & Skipping"]').should('be.visible');
+
+    // check whether all elements of RTP are available
+    e2e.components.ReturnToPrevious.buttonGroup().should('be.visible');
```

Comment:
do we need/care about this extra `buttonGroup` selector? 🤔

Reply:
In this part of the test it may be redundant but I added it for the sake of completeness. In other parts I use the `buttonGroup` selector instead of the single buttons to check whether RTP works as expected.